.drone.yml

```diff
@@ -0,0 +1,59 @@
+---
+kind: pipeline
+type: kubernetes
+name: docs-pr
+
+trigger:
+  event:
+  - pull_request
+
+steps:
+  - name: wait for docker
+    image: docker
+    commands:
+      - timeout 15s /bin/sh -c 'while [ ! -S /var/run/docker.sock ]; do sleep 1; done'
+      - docker version
+    volumes:
+      - name: dockersock
+        path: /var/run
+  - name: build buildbox
+    image: docker:git
+    commands:
+      - apk add --no-cache make
+      - make -C docs bbox
+    volumes:
+      - name: dockersock
+        path: /var/run
+  - name: build
+    image: docker:git
+    commands:
+      - apk add --no-cache make
+      - make -C docs docs
+    volumes:
+      - name: dockersock
+        path: /var/run
+  - name: lint
+    image: docker:git
+    commands:
+      - apk add --no-cache make
```

Comment:
Does drone cache patterns like this? Adding a package as a command in a particular step?

Reply:
There isn't a cache internal to drone.  The right thing to do here would be to build, upload, and use a `docker:git-make` tag to our quay.  This would save us a call to the alpine cdn and ~1s of time.  In my opinion this isn't worth the extra investment.  Though as this pattern gets repeated across more repos, it may be worth revisiting the idea. Thoughts?